RegExp.make.js

```diff
@@ -160,12 +154,9 @@ RegExp.make = (function () {
     /** @type {!Array.<number>} */
     const ranges = this.ranges;
     /** @type {number} */
-    const n = ranges.length;
-    var pastLastRight = 0;
+    let pastLastRight = 0;
```

Comment:
If you're using `const` anywhere, then you should be using transpilation, since the semantics of `let` and `const` in older Firefox/Chrome versions aren't correct. If you're not using transpilation, then `const` shouldn't be used at all.